apps/job-service/src/jobs/delayed-jobs.controller.ts

```diff
@@ -90,25 +96,34 @@ export class DelayedJobsController {
         uuid: { [Op.in]: jobUpdates.map(({ uuid }) => uuid) },
         createdBy: authenticatedUserId,
         status: { [Op.ne]: "pending" }
-      }
+      },
+      logging: console.log,
```

Comment:
This shouldn't be needed on these queries, and should not go to production.

Reply:
the logs is removed